src/blocks/Share/__stories__/Share.stories.tsx

```diff
@@ -1,25 +1,43 @@
-import {Lang as UIKitLang, configure as uikitConfigure} from '@gravity-ui/uikit';
 import {Meta, StoryFn} from '@storybook/react';
 
+import {blockTransform} from '../../../../.storybook/utils';
 import {PageConstructor} from '../../../containers/PageConstructor';
-import {ShareBLockModel, ShareBlockProps} from '../../../models';
+import {PCShareSocialNetwork, ShareBLockModel} from '../../../models';
 import Share from '../Share';
 
 import data from './data.json';
 
 export default {
     component: Share,
     title: 'Blocks/Share',
+    argTypes: {
+        title: {
+            control: {type: 'text'},
+        },
+        items: {
+            control: {type: 'check'},
+            options: Object.values(PCShareSocialNetwork),
+        },
+    },
 } as Meta;
 
-uikitConfigure({lang: UIKitLang.En});
-
 const DefaultTemplate: StoryFn<ShareBLockModel> = (args) => (
-    <PageConstructor content={{blocks: [args]}} />
+    <PageConstructor content={{blocks: [blockTransform(args)]}} />
```

Comment:
Let's remove the PageConstructor here and just make a Share component. Most likely, this will fix the problem with the controls.

Reply:
Removed PageConstructor